test/unit/cpu/test-reductions.cpp

```diff
@@ -328,45 +334,32 @@ TYPED_TEST_P(NestedReductionCorrectnessTest, NestedReduceSum)
 REGISTER_TYPED_TEST_CASE_P(NestedReductionCorrectnessTest, NestedReduceSum);
 
 #if defined(RAJA_ENABLE_OPENMP)
-using nested_types = ::testing::
-    Types<std::tuple<RAJA::NestedPolicy<RAJA::ExecList<RAJA::seq_exec,
-                                                       RAJA::seq_exec,
-                                                       RAJA::seq_exec> >,
-                     RAJA::seq_reduce>,
-          std::
-              tuple<RAJA::
-                        NestedPolicy<RAJA::
-                                         ExecList<RAJA::
-                                                      omp_collapse_nowait_exec,
-                                                  RAJA::
-                                                      omp_collapse_nowait_exec,
-                                                  RAJA::
-                                                      omp_collapse_nowait_exec>,
-                                     RAJA::OMP_Parallel<> >,
-                    RAJA::omp_reduce>,
-          std::
-              tuple<RAJA::
-                        NestedPolicy<RAJA::ExecList<RAJA::omp_parallel_for_exec,
-                                                    RAJA::seq_exec,
-                                                    RAJA::seq_exec> >,
-                    RAJA::omp_reduce>,
-          std::
-              tuple<RAJA::
-                        NestedPolicy<RAJA::
```

Comment:
Turns out that the difference is that I'm using clang-format 5.0.0, which is the current version on homebrew.  Maybe we should try and standardize on that since the difference is no noticeable?